.github/workflows/codechecker-analysis.yml

```diff
@@ -14,9 +14,9 @@ on:
       - fix/**
       - improvement/**
       - wip
-    pull_requests:
-      branches:
-        - master
+  pull_request:
```

Comment:
This was broken in 820f8973. The syntax is `pull_request` and not `pull_request`**`s`**. Also, the indentation was bad. :grin: 

Fun fact: there's a coc plugin for checking JSON schemae or something, because my editor was showing an error:
![Property pull_requests is not allowed](https://user-images.githubusercontent.com/1969470/120886943-b0904580-c5f0-11eb-9e86-0c6a1d451cbd.png)